xml2rfc_compat/tests/test_serializers.py

```diff
@@ -712,6 +907,11 @@ def test_fail_extract_3gpp_tr_series(self):
         docid = DocID(id=id_value, type="TYPE")
         self.assertIsNone(extract_3gpp_tr_series(docid))
 
+    def test_fail_extract_3gpp_tr_series_invalid_prefix(self):
```

Comment:
NIT: Might be worth adding a docstring-type comment explaining a bit about what this test does similar to `test_fail_extract_3gpp_tr_series`.